forge-game/src/main/java/forge/game/card/CounterEnumType.java

```diff
@@ -56,6 +56,8 @@ public enum CounterEnumType {
     BLOOD("BLOOD", 255, 108, 111),
 
     BLOODLINE("BLDLN", 224, 44, 44),
+    
+    BLOODSTAIN("BLDT", 224, 44, 44),
```

Comment:
I think BLDST... it's weird for the first letter of the compound word to not be there